api/README.md

```diff
@@ -41,106 +41,58 @@ For local development, you can copy .env.example to .env.
 NOTE: matching the version of postgresql, postgis and gdal with production is problematic, and best
 avoided. (postgresql + postgis binaries on mac use a newer version of gdal that we don't have on debian yet.)
 
-NOTE: installing postgresql, postgis and gdal as binaries is the preffered method of installation,
-instructions for installing from source are included for reference.
-
-##### Postgresql - from source
-
-```bash
-./configure
-make
-sudo make install
-
-# create a user for postgres
-sudo dscl . -create /Users/postgres
-sudo dscl . -create /Users/postgres UserShell /bin/bash
-sudo dscl . -create /Users/postgres RealName Postgresql
-sudo dscl . -create /Users/postgres UniqueID `dscl . -list /Users UniqueID | awk '{print $2 + 1}' | sort -n | tail -1`
-sudo dscl . -create /Users/postgres NFSHomeDirectory /usr/local/pgsql
-
-# grant user rights, create database
-sudo chown postgres /usr/local/pgsql/data
-sudo -u postgres /usr/local/pgsql/bin/initdb -D /usr/local/pgsql/data
-
-# start the server
-sudo -u postgres /usr/local/pgsql/bin/pg_ctl -D /usr/local/pgsql/data start
-
-# stop the server
-sudo -u postgres /usr/local/pgsql/bin/pg_ctl -D /usr/local/pgsql/data stop
-```
-
-##### Gdal - from source
-
-Currently our API runs on a version of Debian that only has 2.4.\* versions of gdal available in stable.
-
-Unfortunately brew can't install version 2.4.4. so it has to be manually installed.
-
-required by: Postgis, gdal-python
-
-```bash
-wget https://download.osgeo.org/gdal/2.4.4/gdal-2.4.4.tar.gz
-tar -xzf gdal-2.4.4.tar.gz
-cd gdal-2.4.4
-./configure
-make
-make install
-```
-
-##### Gettext - from source
-
-required by: Postgis
+NOTE: installing postgresql, postgis and gdal as binaries is the preffered method of installation. Installing
+from source is a world of trouble you don't want to get into. Stick to brew.
 
-```bash
-wget https://ftp.gnu.org/pub/gnu/gettext/gettext-0.20.2.tar.gz
-tar -xzf gettext-0.20.2.tar.gz
-cd gettext-0.20.2
-./configure
-make
-make install
-```
+##### Java
 
-##### Postgis - from source
+Some of the unit tests use jnius to compare output against RedAPP. The default version of Java that
+comes with mac is likely to cause Segmentation Errors when you run unit tests.
 
-Installing postgis with brew, will break gdal, so it too has to be installed from source.
-
-requires: gettext
+Install the latest JDK! Download [https://jdk.java.net/](https://jdk.java.net/)
 
 ```bash
-wget https://download.osgeo.org/postgis/source/postgis-3.0.1.tar.gz
-tar -xzf postgis-3.0.1.tar.gz
-cd postgis-3.0.1
-./configure
-make LDFLAGS="-L/usr/local/opt/gettext/lib" CPPFLAGS="-I/usr/local/opt/gettext/include"
-sudo make install
+tar -xf openjdk-16.0.2_osx-x64_bin.tar.gz
+sudo mv jdk-16.0.2.jdk /Library/Java/JavaVirtualMachines
 ```
 
-##### Gdal - from brew
+##### Gdal
 
 ```bash
 brew install gdal
-
 ```
 
 ##### Poetry
 
+Try to match the latest version of python in out production environment (as of writing, API is on 3.8.6
```

Comment:
Is `poetry` choosing the minor number differently when running API vs C-Haines?

Reply:
Poetry is grabbing whatever the available python version is on the OS. API (debian) has a different version from C-Haines (ubuntu);

Reply:
Ideally we should really get rid of the Ubuntu version! Once Debian has a new version of GDAL we can do that easily.